TestFoundation/TestNSBundle.swift

```diff
@@ -115,12 +115,12 @@ class TestNSBundle : XCTestCase {
             
             // Put some resources in the bundle
             for n in _bundleResourceNames {
-                NSFileManager.defaultManager().createFile(atPath: bundlePath + "/" + n, contents: nil, attributes: nil)
+                _ = NSFileManager.defaultManager().createFile(atPath: bundlePath + "/" + n, contents: nil, attributes: nil)
```

Comment:
I wonder if we should annotate this API as a discardable result instead. It doesn't have the error handling anyway...